release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -277,6 +287,12 @@ For more information, see xref:../authentication/managing_cloud_provider_credent
 
 // Note: use [discrete] for these sub-headings.
 
+[discrete]
+[id="ocp-4-9-security-etcd-auto-defrag"]
+==== Automatic defragmentation for etcd data
+
+In {product-title} 4.9, etcd data is automatically defragmented, as an alternative to manual defragmentation. Automatic defragmentation is performed by the etcd Operator and is the preferred defragmentation method.
```

Comment:
"as an alternative to manual defragmentation" makes it sound like you can choose to do manual defrag as opposed to auto defrag. Am I correct in assuming that you can't turn off auto defrag? If so, then I'd consider rewording or removing that part, and maybe also the "preferred defragmentation method".

Do you by chance know if it defrags at a set interval (like how history compaction is done every 5 minutes)? If so, might be worth mentioning. No worries if not.

Reply:
I talked to Lilli about this and she said that manual defrag is not needed/required anymore. I talked to Sam B and Eric Rich and they recommend leaving the information about the manual defrag in the doc for now. Also, users cannot modify anything about the auto defrag just yet and the auto defrag is a time-based trigger that happens on a set interval by the Operator - that interval is not known at this time..

Reply:
This content is in the **Notable technical changes** section, and although there is a new etcd section, I still think it should remain here, thoughts?